src/config/jwt.config.ts

```diff
@@ -0,0 +1,20 @@
+import { ConfigModule, ConfigService } from "@nestjs/config";
+import { type JwtModuleAsyncOptions } from "@nestjs/jwt";
+
+export const jwtAccessConfig: JwtModuleAsyncOptions = {
+  imports: [ConfigModule],
+  inject: [ConfigService],
+  useFactory: (configService: ConfigService) => ({
+    secret: configService.get("jwtSecret.access"),
+    signOptions: { expiresIn: "1d" },
+  }),
+};
+
+export const jwtAccessConfigJest: JwtModuleAsyncOptions = {
+  imports: [ConfigModule],
+  inject: [ConfigService],
+  useFactory: (configService: ConfigService) => ({
+    secret: configService.get("jwtSecret.access"),
+    signOptions: { expiresIn: "1d" },
+  }),
+};
```

Comment:
Seems like they're the same. althouth it's good to preserve the possibility to extend, but I would prefer to not have this, create when needed.

Reply:
Agree.
[refactor: delete jwtAccessConfigJest use by jwtAccessConfig](https://github.com/Cophr/backend/pull/10/commits/285e343a348796c932612de940000820bac15622)